src/org/thoughtcrime/securesms/qr/ScanningThread.java

```diff
@@ -114,7 +114,7 @@ public void stopScanning() {
 
       if (result != null) return result.getText();
 
-    } catch (NullPointerException | ChecksumException | FormatException e) {
+    } catch (NullPointerException | ChecksumException | FormatException | IndexOutOfBoundsException e) {
```

Comment:
Do you know what was causing the exception?

Reply:
```
java.lang.ArrayIndexOutOfBoundsException: 
  at com.google.zxing.common.HybridBinarizer.calculateBlackPoints (HybridBinarizer.java:199)
  at com.google.zxing.common.HybridBinarizer.getBlackMatrix (HybridBinarizer.java:79)
  at com.google.zxing.BinaryBitmap.getBlackMatrix (BinaryBitmap.java:85)
```
Seems to be the QR library itself :(